src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -286,8 +286,11 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
 
     val proxyGroupDeletion = userF.flatMap(gcsDAO.deleteProxyGroup)
 
+    val tokenRemoval = toFutureTry(gcsDAO.revokeToken(userRef) flatMap { _ => gcsDAO.deleteToken(userRef) })
```

Comment:
we should not use this elsewhere just for the sake of consistency because it has functional impact. We should only do this if we want to handle errors differently which I don't think we do in other cases.